pkg/cmd/clustertask/describe.go

```diff
@@ -134,6 +139,20 @@ or
 				Err: cmd.OutOrStderr(),
 			}
 
+			if err := validate.NamespaceExists(p); err != nil {
```

Comment:
`ClusterTask` do not require a `namespace` :angel:

Reply:
Yes, this was taken out in #547